File: Java/src/test/java/org/codecop/dependencies/HelloTest.java
package org.codecop.dependencies;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class HelloTest {

    @Test
    public void shouldGreet() {
        Hello hello = new Hello();

        String greeting = hello.greeting();

        assertEquals("Hello World", greeting);
    }

}

File: PHP/test/HelloTest.php
<?php

namespace Org\Codecop\Dependencies\Tests;

use PHPUnit\Framework\TestCase;
use Org\Codecop\Dependencies\Hello;

class HelloTest extends TestCase {

    /**
     * @test
     */
    public function shouldGreet() {
        $hello = new Hello();

        $greeting = $hello->greeting();

        $this->assertEquals("Hello World", $greeting);
    }

}
